pkg/daemon/daemon.go

```diff
@@ -363,18 +341,16 @@ func (dn *Daemon) syncNode(key string) error {
 		glog.V(4).Infof("Finished syncing node %q (%v)", key, time.Since(startTime))
 	}()
 
-	// Handle initial setup
-	if dn.booting {
-		if err := dn.bootstrapNode(); err != nil {
-			return errors.Wrapf(err, "during bootstrap")
-		}
-		return nil
-	}
-
 	_, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
 		return err
 	}
+	// If this isn't our node, nothing to do.  The node controller
+	// handles other nodes.
+	if name != dn.name {
+		return nil
```

Comment:
we're still going to get a bunch of "Started syncing node <other-node>.. \nFinished syncing node <other-node>..." in the logs. Unless we're filtering earlier and this would be unexpected in which case we should log something about it.

Reply:
These aren't printed at our default log level.  I don't think I've ever turned it up beyond the non-default myself honestly, I think there's...a *lot* of spam.